### DISCUSSION TITLE
Support "@link" jsdoc comments

### DISCUSSION THREAD

**Discussion by IanVS** [storybookjs/storybook#24803] [Ideas]
### Is your feature request related to a problem? Please describe.

When using [@link](https://jsdoc.app/tags-inline-link) jsdoc comments, Storybook docs does not apply any formatting nor does it strip the syntax.  So, the Docs view is something like this:

<img width="716" alt="image" src="https://github.com/storybookjs/storybook/assets/4616705/a4ba073f-9a56-4484-a226-acbbbc922b59">

### Describe the solution you'd like

I think the first option could be to just strip the syntax and only show the text without the link notation.

Alternatively, some fancy actual linking could happen, though I'm not sure what I would want the behavior to be, since these are links to source files, and not links to stories.  

### Are you able to assist to bring the feature to reality?

no

### Additional context

We use these types of comments so that TypeScript renames will automatically update the comments as well as the code.

I don't really use addon-docs currently, but might in the future.

**Comment by matthewmatician** [comment]
I would also like to support other jsdoc tags, like `@author` and `@summary`.

**Comment by michaelmcnees** [comment]
Using Markdown syntax for links will work in a JSDoc comment in a story file. It would be nice to have proper JSDoc support if that is the recommended way to add these descriptions (which the docs say to use).
